buildSrc/src/main/java/org/elasticsearch/gradle/internal/rest/compat/RestCompatTestTransformTask.java

```diff
@@ -105,9 +107,40 @@ public void replaceMatch(String subKey, Object value) {
         transformations.add(new ReplaceMatch(subKey, MAPPER.convertValue(value, JsonNode.class)));
     }
 
+    /**
+     * Replaces the values of a match assertion for the given REST test. For example "match":{"_type": "foo"} to "match":{"_type": "bar"}
+     *
+     * @param subKey   the key name directly under match to replace. For example "_type"
+     * @param value    the value used in the replacement. For example "bar"
+     * @param testName the testName to apply replacement
+     */
+    public void replaceMatch(String subKey, Object value, String testName) {
```

Comment:
nit: can you rename this (and the associated class to replaceValueInMatch ? to help distinguish the difference between this and replaceKeyInMatch.

Reply:
I think f64538ab84316a01ea4f5dc406b1510ae06b7fc2 should cover what you're asking for here.

Reply:
@joegallo - thanks! LGTM